polaris-migrator/codeshift.config.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
       'styles-insert-stylelint-disable',
     ),
     'styles-replace-custom-property': resolve('styles-replace-custom-property'),
+    'styles-replace-inline-comments': resolve('styles-replace-inline-comments'),
```

Comment:
```suggestion
```

Correct me if I'm wrong @sam-b-rose but we're not using presets right now, right? Tbh, I didn't even realize this file still existed after the consolidation of `polaris-codemods` and `polaris-migrator`.

Reply:
I left this in as a just-in-case we wanted to try to use something like `@codeshift/cli`; however, I think we should delete this file. I think this change safe to be included in this PR.